chain/src/Pos/Chain/Txp/Tx.hs

```diff
@@ -96,12 +102,15 @@ instance NFData Tx
 txF :: Format r (Tx -> r)
 txF = build
 
+{-# ANN module ("HLint: ignore Reduce duplication" :: Text) #-}
```

Comment:
Is this necessary? If so, does this need to be a module annotation? Maybe we can be more specific and use a function annotation. This isn't a big deal, though.

Reply:
I tried to make it function specific but that didn't work :(